packages/tokens/config.js

```diff
@@ -130,6 +138,15 @@ module.exports = {
             noSort: true,
           },
         },
+        {
+          destination: 'types/font/styles.d.ts',
+          format: 'typescript/es6-declarations/composite',
+          filter: 'filter/font/composite-npm',
+          options: {
+            exportName: 'styles',
+            noSort: true,
```

Comment:
Why is the typing not sorted while the JS is? Seems like they should be consistent. 

Given their composite nature not giving a clear numeric/size hierarchy, I think sorting both is fine?